[Bug] bulma-ui semantic-release still triggers on create-bestax breaking changes (PR #120 incomplete)

## Affected Package(s)

Which package(s) in the monorepo does this bug relate to?

- [x] bulma-ui (`@allxsmith/bestax-bulma`)
- [ ] create-bestax (`create-bestax`)
- [ ] docs (`@allxsmith/bestax-docs`)
- [ ] Other (please specify):

---

## Description

PR #120 attempted to fix the bulma-ui semantic-release configuration but the rules weren't comprehensive enough. The bulma-ui package is still attempting to version bump when create-bestax has breaking changes, despite having ignore rules in place.

The Angular preset's default breaking change behavior is taking precedence over the custom scope rules.

---

## Reproduction Steps

Steps to reproduce the behavior:

1. PR #118 merged with `feat(create-bestax)!:` breaking change commit
2. PR #120 merged with basic ignore rules for create-bestax scope
3. CI runs semantic-release for bulma-ui
4. bulma-ui still attempts to bump to v3.0.0 despite ignore rules
5. NPM version command fails due to dependency conflicts

---

## Expected Behavior

bulma-ui semantic-release should completely ignore ALL commits scoped to create-bestax, including breaking changes. Only commits with `bulma-ui` scope should trigger bulma-ui releases.

---

## Actual Behavior

bulma-ui semantic-release is still analyzing the breaking change commit from create-bestax and attempting a major version bump to v3.0.0, which causes CI failures.

---

## Screenshots / Code

Error from CI (after PR #120):
```
[semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: feat(create-bestax)!: require Node.js 18+ and align with bestax-bulma v2
[semantic-release] › The next release version is 3.0.0
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
```

Current rules in bulma-ui/release.config.js (from PR #120):
```javascript
releaseRules: [
  { scope: 'create-bestax', release: false },
  { breaking: true, scope: 'create-bestax', release: false },
  // ... other rules
]
```

---

## Environment

- Package version(s): bulma-ui 2.6.2, create-bestax 1.4.2
- React version: 18.3.1
- Browser (if applicable): N/A
- OS: GitHub Actions Ubuntu
- Node version: 20.x
- Other relevant info: Monorepo with independent versioning, semantic-release with Angular preset

---

## Additional Context

PR #120's fix was insufficient because:
1. The Angular preset processes breaking changes before custom scope rules
2. Need explicit rules for ALL commit types with create-bestax scope
3. Need catch-all wildcard rules to prevent any non-bulma-ui commits from triggering releases
4. Need final catch-all for unscoped breaking changes

The fix requires comprehensive rules that override the Angular preset's defaults.

--- a/bulma-ui/release.config.js
+++ b/bulma-ui/release.config.js
@@ -7,27 +7,38 @@ export default {
       {
         preset: 'angular',
         releaseRules: [
-          // Explicitly ignore create-bestax scoped commits (including breaking changes)
-          { scope: 'create-bestax', release: false },
+          // FIRST: Explicitly ignore ALL create-bestax commits (must be first to take precedence)
+          { type: 'feat', scope: 'create-bestax', release: false },
+          { type: 'fix', scope: 'create-bestax', release: false },
+          { type: 'docs', scope: 'create-bestax', release: false },
+          { type: 'style', scope: 'create-bestax', release: false },
+          { type: 'refactor', scope: 'create-bestax', release: false },
+          { type: 'perf', scope: 'create-bestax', release: false },
+          { type: 'test', scope: 'create-bestax', release: false },
+          { type: 'chore', scope: 'create-bestax', release: false },
+          { type: 'build', scope: 'create-bestax', release: false },
+          { type: 'ci', scope: 'create-bestax', release: false },
           { breaking: true, scope: 'create-bestax', release: false },
+          { scope: 'create-bestax', release: false }, // Catch-all for create-bestax
 
-          // Release only for bulma-ui scoped commits (independent versioning)
+          // Release ONLY for bulma-ui scoped commits
           { type: 'feat', scope: 'bulma-ui', release: 'minor' },
           { type: 'fix', scope: 'bulma-ui', release: 'patch' },
           { type: 'perf', scope: 'bulma-ui', release: 'patch' },
           { type: 'refactor', scope: 'bulma-ui', release: 'patch' },
           { type: 'style', scope: 'bulma-ui', release: 'patch' },
-
-          // Breaking changes for bulma-ui
           { breaking: true, scope: 'bulma-ui', release: 'major' },
 
-          // Explicitly ignore docs and other scopes
+          // Ignore all other scopes and types (no release unless explicitly bulma-ui scoped)
           { scope: 'docs', release: false },
+          { scope: '*', release: false }, // Ignore any other scope
           { type: 'docs', release: false },
           { type: 'chore', release: false },
           { type: 'ci', release: false },
           { type: 'test', release: false },
           { type: 'build', release: false },
+          { type: '*', release: false }, // Ignore any other type
+          { breaking: true, release: false }, // Ignore all other breaking changes
         ],
       },
     ],

